velox/functions/lib/string/StringImpl.h

```diff
@@ -419,6 +419,15 @@ FOLLY_ALWAYS_INLINE bool fromBase64(
   return true;
 }
 
+template <typename TOutString, typename TInString>
+FOLLY_ALWAYS_INLINE bool toBase64Url(
```

Comment:
this method can be void

Reply:
Right. I suppose the `fromBase64`/`toBase64` could be changed too. Not sure why they are declared with return type bool now that you point it out.

Reply:
@czentgr Yes, it would be nice to change other functions as well (preferably, in a separate PR). We didn't always support void call functions. These may have been added before we added such support.

Reply:
Sounds good, I can manage with a cleanup PR when also fixing the naming.